plugins/view-allocations.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: view-allocations
+spec:
+  version: "v0.5.0"
+  platforms:
+    - selector:
+        matchLabels:
+          os: darwin
+          arch: amd64
+      uri: https://github.com/davidB/kubectl-view-allocations/releases/download/0.5.0/kubectl-view-allocations_0.5.0-x86_64-apple-darwin.tar.gz
```

Comment:
You should also install a license file. As you are defaulting to the copy-all rule, it will be sufficient to just include the `LICENSE` file in your plugin archive.